cpp/src_prims/common/cuml_comms_int.hpp

```diff
@@ -43,6 +43,8 @@ class cumlCommunicator {
   enum datatype_t { CHAR, UINT8, INT, UINT, INT64, UINT64, FLOAT, DOUBLE };
   enum op_t { SUM, PROD, MIN, MAX };
 
+  static const int ANY_RANK = -1;
```

Comment:
How about exposing this along our cumlcomms interface so that users don't have to redeclare tags like `UCP_ANY_RANK` as done above?

Reply:
I defined a separate constant only to keep `ucp_helper` decoupled from the comms implementation, since I've seen some interest in using it on its own. I agree, though, creating a separate constant does make this awkward and brittle. I'll think about this one a bit. Definitely agree the ANY_RANK constant should be moved to the interface.